crypto/evp/pmeth_lib.c

```diff
@@ -1534,6 +1534,28 @@ const char *EVP_PKEY_CTX_get0_propq(EVP_PKEY_CTX *ctx)
     return ctx->propquery;
 }
 
+const OSSL_PROVIDER *EVP_PKEY_CTX_get0_provider(EVP_PKEY_CTX *ctx)
```

Comment:
Should the parameter be `const EVP_PKEY_CTX *ctx`?

Reply:
No other function does that...

Reply:
_We've always done it that way_ isn't a good excuse.

Reply:
> No other function does that...

Yeah, I've noticed. Is it too late for constification of the other getter functions for 3.0? Adding const in such cases does not break API and could be regarded as a bug fix in some sense.

Reply:
I don't think it's too late but it would probably require vote...
I'd still like to see a beginning here.